core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -805,6 +808,8 @@ private[spark] class MapOutputTrackerMaster(
     shuffleStatuses.get(shuffleId) match {
       case Some(shuffleStatus) =>
         shuffleStatus.updateMapOutput(mapId, bmAddress)
+      case None if shuffleMigrationEnabled =>
+        logDebug(s"Asked to update map output for unknown shuffle ${shuffleId}")
```

Comment:
I am very hesitant with making this debug.
If missing, in non migration context, it expected to be logged as an error - making it debug would essentially suppress that message.

Reply:
How about making it `info` or `warn` level?

Reply:
I am fine with warn.
+CC @dongjoon-hyun as well - since you should be debugging this issue a lot more than me :-)